docs/content/concepts/grpc-overview.mdx

```diff
@@ -567,4 +567,20 @@ python client.py
 
 ## Full node operator configuration
 
-TBD
\ No newline at end of file
+To serve builders using the gRPC API, you must enable gRPC on your Full node by indexing gRPC-specific data. Be aware that during the initial indexing phase, your Full node may not be able to handle other traffic — including existing JSON-RPC requests. Plan your rollout carefully and communicate any downtime to your customers and partners in advance. The time needed for this index sync depends on your node’s hardware and software specifications.
+
+Running gRPC alongside JSON-RPC will increase your Full node’s storage usage. While this is expected in the short term, you will be able to reclaim this storage later by disabling JSON-RPC once it is phased out. Builders are expected to migrate to either gRPC directly, or to [GraphQL RPC](./graphql-rpc.mdx) which would rely on gRPC under the hood in future.
+
+By default, your Full node will retain `14` days of gRPC index data. You can configure this retention window using the following settings in your Full node config:
```

Comment:
This actually isn't a default, idk what the default is, but we should encourage people to configure this so some # (like 14)